apps/browser/lib/hooks/useEvoService.ts

```diff
@@ -179,15 +193,7 @@ export const useEvoService = (
 
     setIsStarting(true);
 
-    if (isAuthenticated) {
-      // Create a ChatID
-      if (!chatId) {
-        chatId = uuid();
-        await createChat(chatId);
-        await handleChatIdChange(chatId);
```

Comment:
This isn't being called anymore, which I think will cause a bug in the app if you switch between chats. Please review and LMK if it indeed does not need to be called anymore.

Reply:
It's called in use effect when chat id changes
The reason it's removed is cause it causes issues when uploading the first file
There's no issue with switching chat, but there is an issue when starting from the "no chatId window" now
I've moved this into draft while I fix it

Reply:
I pushed updates